streamlit_app.py

```diff
@@ -1,38 +1,41 @@
-from collections import namedtuple
-import altair as alt
-import math
-import pandas as pd
-import streamlit as st
+import easyocr as ocr  # OCR
+import streamlit as st  # Web App
+from PIL import Image  # Image Processing
+import numpy as np  # Image Processing
+from easyocr import Reader
 
-"""
-# Welcome to Streamlit!
+# title
+st.title("Easy OCR - Extract Text from Images")
```

Comment:
Wrap the code with function, using global scope isn't good

Reply:
Done